### ISSUE TITLE
Updater position blocks pagination 

### ISSUE THREAD

**Issue by muarachmann** [LibreHealthIO/lh-ehr#1400]
The current position of the updater icon may lead to problems while paginating in the future. This calls for discussion. Your ideas are needed to solve this. Take a look at patients pagination one cant go to the next page 

![screen shot 2019-02-02 at 05 45 29](https://user-images.githubusercontent.com/16350814/52160073-4a0c5d00-26ae-11e9-9edf-ba7a61e1d0de.png)

**Comment by muarachmann** [comment]
@robbyoconnor please for labels for this thanks

**Comment by aethelwulffe** [comment]
Updater icon is not really a good "on by default" item.  Should just be a Administration menu item.  The menu could turn the button on and off visible as a user preference, but better to just have a "hide" option on the button, with a notice that you must re-enable in user prefs, or launch from the Administration menu.  It is dangerous to have it on top of in-use reports and lots of other things.  It interferes with the billing manager and several other things as well..

**Comment by robbyoconnor** [comment]
I agree with Art - the demos are a good example -- I don't want people
updating it...we don't even have the it installed as a git repo -- we
use git but rsync everything over.

**Comment by aethelwulffe** [comment]
Hidden feature.... ON with  "DEV" install type, but not just banging
around with the default install.  "Lite" them should also be changed
from default and dumped as well.  Too many things are goofy with lite.

**Comment by robbyoconnor** [comment]
> Hidden feature.... ON with  "DEV" install type, but not just banging around with the default install.  "Lite" them should also be changed from default and dumped as well.  Too many things are goofy with lite.

This should be server-side as well. Not in the install itself.  I don't want people turning it on in the demo.

**Comment by aethelwulffe** [comment]
Demo is a demo, and we should not in the least be deploying or
configuring anything in the product intended for clinics that base their
entire operations on the product simply for the sake of a demo.  Yes, we
want the upgrade feature easily discoverable, and have a config setting
in the demo database that disables it.  For demos, that means that you
MUST configure it as a demo, and not have the default install be what
makes the LH.io demo happiest.

**Comment by muarachmann** [comment]
ok @aethelwulffe so we get this feature under the admin menu right (under preferences) or some sort of turn of an off feature in the globals. If yes this issue is open for anyone to work on.

**Comment by aethelwulffe** [comment]
Yes

**Comment by muarachmann** [comment]
Okay i will prefer someone take a peek at this

**Comment by tafodinho** [comment]
@muarachmann So ill prefer you hint me more about what is it that needs to be done. Is it just going to be the positioning of the icon or more about the functioning??

**Comment by muarachmann** [comment]
the functionalities are already there. Its just for you to place the updater in a suitable position. First check if user is admin to display this option under the admin menu. down forget the developers mode too and settings menu. Find a way to incorporate it.  Also consider putting this in the `Administration > globals` where it can be turned off and on but should be on by default. Please also look at previous comments

**Comment by tafodinho** [comment]
Looking at where the updater button is positioned right now I think that is the best position. I am just going to focus on turning the button on and off in the Administration > globals

**Comment by muarachmann** [comment]
No @tafodinho look at the issue description i should not be there. Please read the issue comments
